tests/integration/admin/routes/test_templates_integration.py

```
@@ -168,12 +167,12 @@ def test_edit_template_found(mock_app, monkeypatch):
         updated_at=None,
         source=None,
     )
-    monkeypatch.setattr("src.main_app.admin.routes.templates.get_template", lambda id: template)
+    monkeypatch.setattr("src.main_app.admin.routes.templates.TemplateService.get_template", lambda self, id: template)
```

Ruff (0.15.21)

[error] Lambda argument `id` is shadowing a Python builtin

(A006)